packages/bpk-component-chip/src/BpkDismissibleChip.tsx

```diff
@@ -38,32 +40,58 @@ const getClassName = cssModules(STYLES);
 type BpkDismissibleChipProps = Omit<Props, 'disabled' | 'selected'>;
 
 const BpkDismissibleChip = ({
+  accessibilityLabel,
+  children,
   className,
   leadingAccessoryView = null,
   type = CHIP_TYPES.default,
   ...rest
 }: BpkDismissibleChipProps) => {
-  const iconClassNames = getClassName(
-    `bpk-chip--${type}-dismissible__trailing-accessory-view`,
+
+  const classNames = getClassName(
+    'bpk-chip',
+    `bpk-chip--${type}`,
+    `bpk-chip--${type}-dismissible`,
+    `bpk-chip--${type}-selected`,
+    !children && 'bpk-chip--icon-only',
   );
-  const classNames = getClassName(`bpk-chip--${type}-dismissible`, className);
 
   return (
-    <BpkSelectableChip
-      {...rest}
-      leadingAccessoryView={leadingAccessoryView}
-      disabled={false}
-      trailingAccessoryView={<CloseCircleIconSm 
-        // TODO: className to be removed
-        // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-        className={iconClassNames} />}
-      selected
-      type={type}
-      role="button" // Override role="checkbox" because this chip is not selectable.
-      // TODO: className to be removed
-      // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-      className={classNames}
-    />
+    <div className={getClassName(className)}>
+      <button
```

Comment:
best to re-create the BpkSelectableChip as we'd have to pass a classname a wrapping element on an already wrapped Component.
```html
<div classname="custom_classname">
  <div classname="">
    <button classname="bpk_chip">
```